docs/source/course/index.md

```diff
@@ -4,6 +4,11 @@ If you like to learn from video, you can follow our [hands-on course "Introducti
 
 The course is structured into sections and these are each broken into short videos that cover specific Kedro topics. You'll walk through the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) and get hands-on with the example. Along the way, you'll learn key Kedro concepts like datasets and the Kedro Data Catalog, nodes and pipelines, and configuration management.
 
+```{eval-rst}
+..  youtube:: DD7JuYKp6BA
+    :width: 100%
+```
```

Comment:
Nit: This could be MyST

````suggestion
```{youtube} DD7JuYKp6BA
:width: 100%
```
````

(tested locally)

Reply:
I'd prefer it wasn't MyST as I think it's far better for maintainability to follow the exact guidance on their docs than to make a further leap and convert it. We replicate their docs so it's easy to understand what's going on.